src/games/entities.ts

```diff
@@ -7,7 +7,7 @@ export type Board = [ Row, Row, Row, Row, Row, Row,Row, Row, Row, Row]
 type Status = 'pending' | 'started' | 'finished'
 
 //const emptyRow: Row = [null, null, null]
-const emptyBoard: Board = [
+export const emptyBoard: Board = [
```

Comment:
makes lots of sense to put that here. But when and where do we say that we have two of those then?

Reply:
Because when creating a new game, the two boards will be exactly the same. So we can just give them the value from that new game. For example in the database we can store like {board1:newGame(), board2:newGame()}

Reply:
ah, I see. so when we create a new game we have to make sure to create two new identical boards but they will go to the frontend with their names already then?

Reply:
no, I think we don't need to, we can just send one and make a copy in the frontend. Well we'll see ...  :sweat_smile: